grove/grovetccfg/grovetccfg.go

```diff
@@ -35,7 +35,7 @@ import (
 	"time"
 
 	"github.com/apache/trafficcontrol/lib/go-tc"
-	to "github.com/apache/trafficcontrol/traffic_ops/client"
+	to "github.com/apache/trafficcontrol/traffic_ops/v2-client"
```

Comment:
Why is this necessary? Shouldn't we get everything to work w/ v3 since it's simply a copy of v2 at this point?

Reply:
Although, now that I'm recalling the discussions we've had recently about components speaking two major versions of the TO API, maybe TM and TS _should_ stay on API v2 for the 5.0 release. Although, I'm pretty sure at least TM is going to need v3, right, because of dual-homing changes? So maybe it needs to use v2 for most things then _attempt_ v3 for things it needs for new features but still be able to fall back to v2?

Reply:
Yeah, honestly I think upgrading all the clients to v2 right away was probably a mistake. Might let us get rid of Perl sooner, though, so well worth it

> _"use v2 for most things then attempt v3"_

That's an option, but tbh it depends on what you mean by "needs". TM changes will be capable of reading either 2.0 or 3.0 response payloads, so TM will continue to function with 2.0 for the forseeable future. But then it won't be able to provide the new feature.

Reply:
Alright, so it sounds like we keep TM at v2 for the time being, but we'll need to figure something out for it soon in order to take advantage of new 3.0 features.